schemas/metadata/error/error.1.schema.json

```diff
@@ -48,6 +52,7 @@
       "type": "string"
     },
     "host": {
+      "description": "Hostname part of the submission URI, i.e. \"incoming.telemetry.mozilla.org\"",
```

Comment:
I'm not sure if this is technically accurate in all cases (I've seen Host header and URI mismatches before) but it probably doesn't matter.